From 705397f479f758a6eae767aad84e13b10ca25f46 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@gentoo.org>
Date: Wed, 23 Feb 2011 15:49:00 +0000
Subject: Fix header collisions between clutter-gtk-0.10 and clutter-gtk-1.0

Always add CLUTTER_GTK_CFLAGS before CLUTTER_CFLAGS to maintain the path order.
Use the same LIBS order for consistency.

https://bugzilla.gnome.org/show_bug.cgi?id=643080
---
diff --git a/aisleriot/Makefile.am b/aisleriot/Makefile.am
index 9bd5995..030927f 100644
--- a/aisleriot/Makefile.am
+++ b/aisleriot/Makefile.am
@@ -167,9 +167,9 @@ sol_clutter_SOURCES += \
 endif
 
 sol_clutter_CPPFLAGS = $(sol_CPPFLAGS) -DHAVE_CLUTTER
-sol_clutter_CFLAGS = $(sol_CFLAGS) $(CLUTTER_CFLAGS) $(CLUTTER_GTK_CFLAGS)
+sol_clutter_CFLAGS = $(sol_CFLAGS) $(CLUTTER_GTK_CFLAGS) $(CLUTTER_CFLAGS)
 sol_clutter_LDFLAGS = $(sol_LDFLAGS)
-sol_clutter_LDADD = $(sol_LDADD) $(CLUTTER_LIBS) $(CLUTTER_GTK_LIBS)
+sol_clutter_LDADD = $(sol_LDADD) $(CLUTTER_GTK_LIBS) $(CLUTTER_LIBS)
 
 
 endif # ENABLE_AISLERIOT_CLUTTER
diff --git a/gnibbles/Makefile.am b/gnibbles/Makefile.am
index c332c5d..cf0e4ff 100644
--- a/gnibbles/Makefile.am
+++ b/gnibbles/Makefile.am
@@ -39,15 +39,15 @@ gnibbles_CPPFLAGS = \
 
 gnibbles_CFLAGS = \
 	$(GTK_CFLAGS) \
-	$(CLUTTER_CFLAGS) \
 	$(CLUTTER_GTK_CFLAGS) \
+	$(CLUTTER_CFLAGS) \
 	$(AM_CFLAGS)
 
 gnibbles_LDADD = \
 	$(top_builddir)/libgames-support/libgames-support.la \
 	$(GTK_LIBS) \
-	$(CLUTTER_LIBS) \
 	$(CLUTTER_GTK_LIBS) \
+	$(CLUTTER_LIBS) \
 	$(INTLLIBS)
 
 if HAVE_GNOME
diff --git a/quadrapassel/Makefile.am b/quadrapassel/Makefile.am
index dedd075..087d34b 100644
--- a/quadrapassel/Makefile.am
+++ b/quadrapassel/Makefile.am
@@ -31,14 +31,14 @@ quadrapassel_CPPFLAGS = \
 
 quadrapassel_CXXFLAGS = \
 	$(GTK_CFLAGS) \
-	$(CLUTTER_CFLAGS) \
 	$(CLUTTER_GTK_CFLAGS) \
+	$(CLUTTER_CFLAGS) \
 	$(AM_CXXFLAGS)
 
 quadrapassel_LDADD = \
 	$(top_builddir)/libgames-support/libgames-support.la \
-	$(CLUTTER_LIBS) \
 	$(CLUTTER_GTK_LIBS) \
+	$(CLUTTER_LIBS) \
 	$(GTK_LIBS) \
 	$(INTLLIBS)
 
--
cgit v0.8.3.4
